app/resources/v1/users.js

```diff
@@ -98,6 +98,52 @@ exports.post = function (req, res) {
     User.findOne({ twitter_id: twitterCredentials.userId }, handleFindUser)
   } // END function - handleTwitterSignIn
 
+  const handleEmailSignIn = async function (credentials) {
+    function generateId (nickname) {
+      // TODO - Check if the Id generated is not existing
+      return nickname + random(0, 999)
```

Comment:
Can use this format:

`username-1234`

(four digit number with a dash `-` as a separator)

Reply:
Yeah sure